src/style.css

```diff
@@ -14,6 +14,7 @@
   --hover-boxShadow: var(--app-hover-boxShadow);
   --hover-opacity: var(--app-hover-opacity);
   --hover-textDecoration: var(--app-hover-textDecoration);
+  --hover-borderRight: var(--app-hover-borderRight);
```

Comment:
TOL: I wonder if there's a performance cost to adding more of these. In other words, should be be judicious about adding more?

Reply:
I doubt it; it's only adding one css rule to the stylesheet that's loaded, and only elements that change this particular style will have the variable change as it's applied to them, otherwise it's `initial`.